src/main/java/co/cask/gcp/bigquery/BigQuerySink.java

```diff
@@ -159,10 +157,13 @@ public void initialize(BatchRuntimeContext context) throws Exception {
 
   @Override
   public void transform(StructuredRecord input, Emitter<KeyValue<JsonObject, NullWritable>> emitter) throws Exception {
-    List<Schema.Field> fields = config.getSchema().getFields();
+    Schema outputSchemaFields = config.getSchema();
```

Comment:
it's more performant to call this once at initialize time. Generates less garbage in memory, as getSchema() actually creates a new object each time.